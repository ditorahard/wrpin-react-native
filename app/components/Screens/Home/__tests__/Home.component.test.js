import 'react-native';
import React from 'react';
import HomeScreen from '../Home.component';
import renderer from 'react-test-renderer';

it('Home renders correctly', () => {
  const tree = renderer.create(
    <HomeScreen />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});