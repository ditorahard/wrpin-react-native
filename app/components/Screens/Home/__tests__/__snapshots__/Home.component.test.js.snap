// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#D20202",
        "paddingBottom": 20,
        "paddingTop": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 20,
          "fontWeight": "700",
        }
      }
    >
      NEWS
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "elevation": 1,
        "marginBottom": 3,
        "marginTop": 3,
        "padding": 5,
        "shadowColor": "grey",
        "shadowOffset": Object {
          "height": 5,
          "width": 5,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 10,
      }
    }
  >
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "borderBottomWidth": 1,
            "borderColor": "#D9D5DC",
            "borderLeftWidth": 1,
            "borderRadius": 30,
            "borderRightWidth": 1,
            "borderTopWidth": 1,
            "borderWidth": 1,
            "flexDirection": "row",
            "marginLeft": 2,
            "opacity": 1,
          }
        }
      >
        <Text
          active={true}
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 12,
              },
              Array [
                Object {
                  "color": "#000",
                  "fontSize": 30,
                },
                Object {
                  "color": "#000",
                  "fontSize": 24,
                  "paddingLeft": 10,
                  "paddingRight": 8,
                },
              ],
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          ï’¥
        </Text>
        <TextInput
          allowFontScaling={true}
          editable={true}
          placeholder="Search . . ."
          placeholderTextColor="#575757"
          style={
            Array [
              Object {
                "color": "#000",
                "flex": 1,
                "fontSize": 17,
                "height": 50,
                "paddingLeft": 8,
                "paddingRight": 5,
                "top": 1.5,
              },
              Object {
                "fontSize": 15,
              },
            ]
          }
          underlineColorAndroid="rgba(0,0,0,0)"
        />
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "height": 1334,
        "padding": 20,
      }
    }
  >
    <Text>
      Silakan ketik untuk mencari berita
    </Text>
  </View>
</View>
`;
